### DISCUSSION TITLE
Klipper instances

### DISCUSSION THREAD

**Discussion by Insektenspray** [Guilouz/Klipper-Flsun-Speeder-Pad#95] [General]
What is the difference between the single instance and the multiple ones?
Is this a matter of how many printers you want to control with the pad?

**Comment by Monsinia** [comment]
looks like they dont reply very quickly here :( my printer is unusable at the moment from following the guide.  I'm starting it over.  but I was curious about the same thing.

**Comment by Fropzs** [comment]
its if u have more then 1 printer :)

**Comment by Guilouz** [comment]
1 instance = 1 printer used
3 instances = 3 printers used in same times
